src/scalarstats.jl

```diff
@@ -552,24 +552,16 @@ of the standard deviation under the assumption that the data is normally distrib
 """
 function mad!(x::AbstractArray;
               center=median!(x),
-              normalize::Union{Bool,Nothing}=true,
-              constant=nothing)
+              normalize::Union{Bool,Nothing}=false)
     isempty(x) && throw(ArgumentError("mad is not defined for empty arrays"))
     c = center === nothing ? median!(x) : center
```

Comment:
Is it documented anywhere and actually useful that `center::Nothing` is supported?